lisa/target.py

```diff
@@ -197,13 +198,14 @@ class used under the hood, and avoids messing up with ``devlib``
 
     CONF_CLASS = TargetConf
     INIT_KWARGS_KEY_MAP = {
+        'kernel_src': ['kernel_src'],
```

Comment:
That's not needed in basic case but will be needed with the `kernel` sublevel (just like the below devlib's `excluded-modules` case).

Reply:
:+1: